cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -501,6 +499,68 @@ func (command *Evaluate) evaluateDocker(ctx *evaluate.Context) (err error) {
 	}
 	parallel.Wait()
 
+	// Copy data from volume back.
+	{
```

Comment:
nit: would be nice to have this volume copying done in a helper method with some tests

can be follow-up

Reply:
The docker tests are already testing the behavior that everything is copied back from the volumes to the temporary test result path.